### DISCUSSION TITLE
Need Business email?

### DISCUSSION THREAD

**Discussion by jjerhub-guest** [okteto/okteto#3428] [Q&A]
I'm trying to create an account to test pilot the software for my org, but I just get an error saying to specify my business email. I see nowhere in my GitHub profile that I can mark my email address as "for business use" and was unable to find any instructions for doing so on a cursory search.

**Comment by jLopezbarb** [comment]
Hi @jjerhub-guest, there is no extra step to set up the business email. It just needs to appear in the email list in the following link: https://github.com/settings/emails

**Comment by jjerhub-guest** [comment]
Then what do you consider a "business" email? One not from common mail
provider domains? If so, why does it matter to you?

It would seem to me that as long as someone can "pay" for the product it
shouldn't matter what email addresses they have attached to their account.

**Reply by jLopezbarb** [reply]
All emails are carefully reviewed, there is an automatic pre-filter to determine whether or not they could be business emails and then we manually look at whether or not they could be suspicious. This prevents attackers mining in okteto cloud adding a better ecosystem for everyone that wants to try/develop in the Okteto platform

**Reply by nitishfy** [reply]
Hi @jjerhub-guest Please take a look at [this](https://community.okteto.com/t/why-we-only-allow-accounts-with-a-business-email-on-cloud-okteto-com/221) for more information on why do we prefer business email.
